blackrock/templates/respiration/temp_leaf.html

```diff
@@ -2,8 +2,7 @@
 {% load extras %}
 {% block tabname %}tab-leaf{% endblock %} 
 
-{% block js %} 
-  {{ block.super }} 
+{% block jstemp %} 
```

Comment:
The `jstemp` block isn't referred to in the parent template - if you're not using the `js` block here, I think you can just remove this `block` tag and its associated `endblock`.